src/snowflake/snowpark/files.py

```diff
@@ -50,19 +53,24 @@ def open(
         file_location: str,
         mode: str = "r",
         is_owner_file: bool = False,
+        *,
```

Comment:
what is this *?

Reply:
Everything after the * is a keyword only arg (ie can't be specified positionally). This mirrors what `override_snowflake_file_impl` does on the server side.